src/client/storage/uploader.ts

```diff
@@ -52,17 +60,20 @@ export const startChunkUpload = async (chunkId: string) => {
         );
 
         // TODO: check status from bundler
-        if (response.data.status !== 'ok') {
+        if (response.data.status !== 'ok' || !response.data.txid) {
             throw new Error(`Chunk ${chunkId} uploading failed: arweave endpoint error: ${
                 JSON.stringify(response.data, null, 2)
             }`);
         }
 
+        const txid = response.data.txid;
```

Comment:
This seems to be a good idea to store txid, but how do we use it? Looks like we don't